src/components/message.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+interface UserMessage {
+  message: string;
+}
+
+const Message = (props: UserMessage): any => {
```

Comment:
Stylistic preference:
I'd rather declare components using function style than => style. I find it more clear, especially with Typescript.

Reply:
Noted, will do! I will keep it as function :)